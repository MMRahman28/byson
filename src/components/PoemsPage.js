import React from 'react';
//import ExpenseForm from './ExpenseForm';
//import {connect } from 'react-redux';
//import { setPoem } from '../actions/poems';
import RandomPoems from './RandomPoems';
import PoemFilters from './PoemFilters';


const PoemsPage = () => (
  <div>
  <PoemFilters />
   <RandomPoems />
   </div>
);
export default PoemsPage;
