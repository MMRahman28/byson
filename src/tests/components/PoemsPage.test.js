import React from 'react';
import { shallow } from 'enzyme';
import PoemsPage from '../../components/PoemsPage';

test('should render PoemsPage correctly', () => {
  const wrapper = shallow(<PoemsPage />);
  expect(wrapper).toMatchSnapshot();
});
