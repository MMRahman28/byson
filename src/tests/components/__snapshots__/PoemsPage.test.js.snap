// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render PoemsPage correctly 1`] = `
<div>
  <React.Memo />
  <React.Memo />
</div>
`;
